about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="En">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="Sergio Valadez">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/main.css">
+  <link rel="stylesheet" href="css/nav.css">
+  <link rel="stylesheet" href="css/style.css">
+
+  <title>Agro-industry 2021</title>
+  <link rel="icon" href="/images/logo3.png">
+</head>
+
+<body>
+  <header class="topnav">
+    <a><img src="/images/ic_github.png" alt="facebook logo"></a>
+    <a><img src="/images/ic_twitter.png" alt="twitters logo"></a>
+    <a href="#">Spanish</a>
+    <a href="#">My Page</a>
+    <a href="#">Log Out</a>
+  </header>
+  <header>
+    <nav>
+      <a href="index.html"><img class="logo" src="/images/logo.svg" alt="logo"></a>
+
+      <ul class="enlaces-menu">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#program">Programs</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html" class="alert">Hempaing</a></li>
+      </ul>
```

Comment:
- [ ] For accessibility reasons I think it will be nice to add a background to the mobile menu. This will help the visitors know what text they are clicking. actually, your mobile menu navigation is mixed with external text.
![image](https://user-images.githubusercontent.com/65068771/148297313-de8b1ee6-61e6-42a4-86ab-04a0315479ee.png)

- [ ] Also on the about page menu we should find a backlink to the home page. check the screenshot below 👇 
![image](https://user-images.githubusercontent.com/65068771/148298181-0a406640-b88f-4f29-a4e8-a7f6e63610c7.png)

Reply:
Totally agree with you. But if we are strictly following the guidelines, they don't say a word about how mobile menu may look and feel. That said, I solve your observation.

Reply:
thanks for making the changes all the same 👍. we have guidelines to follow and sometimes the requirements you have are not worded the same as we receive them. so please understand when we give you changes it's always to help you improve.

Reply:
I got it. All feedback is positive. That menu was awfully lazy and hard to distinguish among the body's text.